salt/runners/venafiapi.py

```diff
@@ -352,13 +433,46 @@ def show_csrs():
         )
     return data.get('dict', {})
 
+def get_zone_id(zone_name):
+    '''
+    Get the zone ID for the given zone name
+
+    CLI Example:
 
-def show_zones(company_id):
+    .. code-block:: bash
+
+        salt-run venafi.get_zone_id default
     '''
-    Show certificate requests for this API key
+    data = salt.utils.http.query(
+        '{0}/zones/tag/{1}'.format(base_url,zone_name),
+        status=True,
+        decode=True,
+        decode_type='json',
+        header_dict={
+            'tppl-api-key': _api_key(),
+        },
+    )
+
+    status = data['status']
+    if str(status).startswith('4') or str(status).startswith('5'):
+        raise CommandExecutionError(
+            'There was an API error: {0}'.format(data['error'])
+        )
+    return data['dict']['id']
+
+
```

Comment:
With our new policy model, it would be very helpful to let the users see the Certificate Policies that have been configured. The output of /v1/certificatepolicies will show all policies that the user has read access to. This can help them understand what types of certificates are available for them to request.

Would you consider adding a new show_policies() function? It would work virtually identically to the show_zones() function in that you would display all policies and their identifiers that have been configured. The salt user can then use the output of show_zones() to see what policies are available on each zone.